src/Ship.cpp

```diff
@@ -669,6 +669,12 @@ void Ship::SetFlightState(Ship::FlightState newState)
 		m_launchLockTimeout = std::min(std::max(2.0, 2.0 * (GetMass() / MASS_LOCK_REFERENCE)), 20.0);
 	}
 
+	if (newState == DOCKED) {
+		m_launchLockTimeout = 0.0;
```

Comment:
Out of curiosity, is it really necessary to zero it out, or just for the sake of consistency?